src/03-modality-agnostic-files.md

```diff
@@ -81,29 +85,105 @@ Template:
 ```Text
 participants.tsv
 participants.json
-phenotype/<measurement_tool_name>.tsv
-phenotype/<measurement_tool_name>.json
 ```
 
-Optional: Yes
+The purpose of this RECOMMENDED file is to describe properties of participants
+such as age, sex, handedness etc.
+In case of single session studies this file has one compulsory column
+`participant_id` that consists of `sub-<label>`, followed by a list of optional
+columns describing participants.
+Each participant MUST be described by one and only one row.
+
+Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
+and `handedness`. We RECOMMEND to use the following values for these three
+columns in case that you use them:
+
+-   `age`: numeric value in years (float or integer value)
+
+-   `sex`: string value indicating phenotypical sex, one of "male", "female",
+    "other"
+
+    -   for "male", use one of these values: `male`, `m`, `M`, `MALE`, `Male`
 
-The purpose of this file is to describe properties of participants such as age,
-handedness, sex, etc. In case of single session studies this file has one
-compulsory column `participant_id` that consists of `sub-<label>`,
-followed by a list of optional columns describing participants. Each participant
-needs to be described by one and only one row.
+    -   for "female", use one of these values: `female`, `f`, `F`, `FEMALE`,
+      ` Female`
+
+    -   for "other", use one of these values: `other`, `o`, `O`, `OTHER`,
+        `Other`
+
+-   `handedness`: string value indicating one of "left", "right",
+    "ambidextrous"
+
+    -   for "left", use one of these values: `left`, `l`, `L`, `LEFT`, `Left`
+
+    -   for "right", use one of these values: `right`, `r`, `R`, `RIGHT`,
+        `Right`
+
+    -   for "ambidextrous", use one of these values: `ambidextrous`, `a`, `A`,
+        `AMBIDEXTROUS`, `Ambidextrous`
+
+Throughout BIDS you can indicate missing values with `n/a` (i.e., "not
+available").
 
 `participants.tsv` example:
 
 ```Text
-participant_id  age sex group
-sub-control01 34  M control
-sub-control02 12  F control
-sub-patient01 33  F patient
+participant_id age sex handedness group
+sub-01 34 M right read
+sub-02 12 F right write
+sub-03 33 F n/a read
+```
+
+It is RECOMMENDED to accompany each `participants.tsv` file with a sidecar
+`participants.json` file to explain the values in the TSV columns (see also
+the [section on tabular files](02-common-principles.md#tabular-files)).
+Such sidecar files are needed to interpret the data, especially so when
+optional columns are defined beyond `age`, `sex`, and `handedness`, such as
+`group` in this example.
+
+`participants.json` example:
```

Comment:
is the rationale for having this sidecar file described elsewhere in the document? In the example below, it's not clear to me how one would write the "Levels". Also, why is this even needed when you already have a restricted set of values or `age` and `sex`?

Reply:
yes I link to a section where this is described in the sentence above `[section on tabular files](02-common-principles.md#tabular-files))`

> Also, why is this even needed when you already have a restricted set of values or age and sex

with this PR, we only RECOMMEND, not REQUIRE --> as I say in the PR intro, the changes here are a first step that should be ok with both sides of the discussion (those that want to have it strict, and those who want to have it lenient or relating to other standards/ontologies)

we need to discuss in the related issue on how to improve things further.

Reply:
okay fair enough. Let's continue the discussion in another issue/PR!